examples/update/package.json

```diff
@@ -1,5 +1,6 @@
 {
     "scripts": {
+        "benchmark": "ts-node --transpile-only --ignore=false benchmark/benchmark.ts",
```

Comment:
The convention has been to put benchmark below test, if we want to change that we should do it across all of the examples